mithril-signer/Cargo.toml

```diff
@@ -37,15 +42,7 @@ mithril-signed-entity-lock = { path = "../internal/signed-entity/mithril-signed-
 mithril-signed-entity-preloader = { path = "../internal/signed-entity/mithril-signed-entity-preloader" }
 mithril-ticker = { path = "../internal/mithril-ticker" }
 rand_core = { workspace = true }
-reqwest = { workspace = true, features = [
-    "default",
-    "stream",
-    "gzip",
-    "zstd",
-    "deflate",
-    "brotli"
-] }
-semver = { workspace = true }
+reqwest = { workspace = true, features = ["default"] }
```

Comment:
Do we still need the `reqwest` dependency in the signer?

Reply:
OK :+1: 

Can we target only the features for HTTP/2 and TLS instead of `default` in that case?

Reply:
I can do that, but that would means that If reqwest change its default feature set, like to add http/3, we would have to enables them manually.